### DISCUSSION TITLE
Disk Cache

### DISCUSSION THREAD

**Discussion by djouallah** [duckdb/duckdb#5055] [General]
I know you don't like talking about long term plan, just wondering, is there any plan to implement disk cache When querying remote cloud storage like S3, the throughput is just too low, and you can't have pleasant experience without a form of Disk cache :(

**Comment by hannes** [comment]
Nothing keeps you from materialising data read from cloud storage locally, no?

**Comment by djouallah** [comment]
yes, but often, you don't query the whole parquet files just some columns, or a subset of rows, currently I Rsync everything to the local SSD but it is far from optimal,

**Comment by hannes** [comment]
How many row groups do your parquet files have?

**Comment by djouallah** [comment]
num_columns: 16
  num_rows: 59986052
  num_row_groups: 600
  format_version: 2.6
  serialized_size: 995199
